src/class-elements/static-private-method-subclass-receiver.case

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2019 Kubilay Kahveci (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Static private methods on the super-class cannot be called with sub-class as the receiver
+info: |
+  PrivateFieldGet (P, O)
+    1. Assert: P is a Private Name.
+    2. If O is not an object, throw a TypeError exception.
+    3. If P.[[Kind]] is "field",
+        ...
+    4. Perform ? PrivateBrandCheck(O, P).
+    5. If P.[[Kind]] is "method",
+      a. Return P.[[Value]].
+    ...
+
+  PrivateBrandCheck(O, P)
+    1. If O.[[PrivateBrands]] does not contain an entry e such that SameValue(e, P.[[Brand]]) is true,
+      a. Throw a TypeError exception.
+template: default
+features: [class-static-methods-private]
```

Comment:
please add public as well

Reply:
I couldn't find a feature tag for static public methods in the codebase and they are not part of https://github.com/tc39/proposal-static-class-features/. Should I just create one in `features.txt`?

Reply:
yes